frontend/www/js/omegaup/components/problem/creator/cases/Sidebar.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <div class="d-flex align-items-center justify-content-between">
+      <h5 class="mb-0 d-none d-md-inline">{{ T.problemCreatorGroups }}</h5>
+      <div>
+        <b-button size="sm" variant="primary" class="mr-2">
+          <BIconLayoutSidebar />
+        </b-button>
+        <b-button size="sm" variant="success" class="mr-2">
+          <span class="d-none d-xl-inline">{{ T.problemCreatorAdd }}</span>
+          <BIconPlusCircle class="d-inline d-xl-none" />
+        </b-button>
+        <b-button variant="light" size="sm">
+          <BIconThreeDotsVertical />
+        </b-button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import T from '../../../../lang';
+@Component({})
```

Comment:
nit:

```suggestion
@Component()
```

esto es equivalente